### PR TITLE
Update missing references and add release notes

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: .github/workflows/deploy_env.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         id: install
         with:
           version: latest
-      - uses: ministryofjustice/hmpps-github-actions/.github/actions/build-test-and-deploy/cloud-platform-deploy@v1 # WORKFLOW_VERSION
+      - uses: ministryofjustice/hmpps-github-actions/.github/actions/build-test-and-deploy/cloud-platform-deploy2 # WORKFLOW_VERSION
```
Comment (Sandhya1874) [inline .github/workflows/deploy_env.yml:32]: Does this need to be @v2 ?

Comment (andrewrlee) [reply .github/workflows/deploy_env.yml:32]: yes!

Comment (andrewrlee) [reply .github/workflows/deploy_env.yml]: Thanks fixed that!

**Review by Sandhya1874 — APPROVED** [review]
LGTM

**Pull request merged**